addons/website_sale/static/src/js/location_selector/location_selector_dialog/location_selector_dialog.xml

```diff
@@ -1,92 +1,208 @@
+                                                        </span>
+                                                        <span class="d-contents">
+                                                            <small class="me-4 text-muted">Thursday</small>
+                                                            <span class="o_locationSelector_schedule_hours d-flex flex-wrap">
+                                                                <small>06:30 - 12:00</small>
+                                                                <small>13:00 - 19:30</small>
+                                                            </span>
+                                                        </span>
+                                                        <span class="d-contents">
+                                                            <small class="me-4 text-muted">Friday</small>
+                                                            <span class="o_locationSelector_schedule_hours d-flex flex-wrap">
+                                                                <small>13:00 - 19:30</small>
+                                                            </span>
+                                                        </span>
+                                                        <span class="d-contents">
+                                                            <small class="me-4 text-muted">Saturday</small>
+                                                            <span class="o_locationSelector_schedule_hours d-flex flex-wrap">
+                                                                <small class="text-danger">Closed</small>
+                                                            </span>
+                                                        </span>
+                                                        <span class="d-contents">
+                                                            <small class="me-4 text-muted">Sunday</small>
+                                                            <span class="o_locationSelector_schedule_hours d-flex flex-wrap">
+                                                                <small class="text-danger">Closed</small>
+                                                            </span>
+                                                        </span>
+                                                    </span>
+                                                </span>
+                                                <!-- TODO: Toggle "Open now/Close now" elements depending on the time -->
+                                                <small class="d-md-none text-success fw-bold">Open now</small>
+                                                <!-- <small class="d-md-none text-danger fw-bold">Close now</small> -->
+                                            </span>
+                                        </button>
+                                    </div>
```

Comment:
IMO this should be a collapse as the first location will always be selected anyway (it should already be the case but maybe there is a bug on my part). Also, to reduce the code, could you move that part to the Map component?

Reply:
Ok I've removed the collapse and moved the code into the map view 👍 .
I also took the opportunity to clean up and further simplify the overall xml structure.

My only concern is that this button doesn't work anymore since I moved it into the map view.
Could you take a look at it? I've left you a few comments in the code 👇 
https://github.com/odoo-dev/odoo/pull/3087/files#diff-137b82b9720f2631064e69e4e15ed7e89bb15ebe7b528af60d9809ff0d6265d3R16-R19